### PR TITLE
NAID-351: Map ITK Report to FHIR List

### PR DESCRIPTION
**Pull request by Broom94**

### CODE REVIEW

File: service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java [hunk 1]
```diff
@@ -47,6 +59,17 @@ public Bundle createEncounterBundle(POCDMT000002UK01ClinicalDocument1 clinicalDo
         addEpisodeOfCare(bundle, encounter);
         addComposition(bundle, composition);
 
+
+        List<Resource> resourcesCreated = bundle.getEntry().stream().map(it -> it.getResource()).collect(Collectors.toList());
+
```
Comment (dawidskainos) [inline service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java:64]: too many empty lines here

File: service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java [hunk 2]
```diff
@@ -166,9 +189,9 @@ private void addComposition(Bundle bundle, Composition composition) {
         }
     }
 
-    private static void addEntry(Bundle bundle, Resource resource) {
+    private void addList(Bundle bundle, ListResource listResource) {
         bundle.addEntry()
```
Comment (dawidskainos) [inline service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java:193]: use generic addEntry method

File: service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java [hunk 3]
```diff
@@ -29,6 +34,13 @@
 
     private EncounterMapper encounterMapper;
     private CompositionMapper compositionMapper;
+    private ListMapper listMapper;
+
+    private static void addEntry(Bundle bundle, Resource resource) {
```
Comment (dawidskainos) [inline service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java:39]: please don't put private methods above public ones

Comment (Broom94) [reply service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java:39]: Code formatter moves it to here, but I can move it back

**Pull request merged**